### ISSUE TITLE
How can I continue training based on the model I've trained ？？？  

### ISSUE THREAD

**Issue by xinliu111** [seasonSH/DocFace#12]

**Comment by seasonSH** [comment]
If you are training the sibling network. Just change "restore_model" in config/finetune.py to your own model folder. Then, comment out all the items in the "replace_scopes". Then run "python src/train_sibling.py config/finetune.py" again and it should work.

**Comment by xinliu111** [comment]
为什么准确率降到这么低？？
Testing...
[1] TAR: 0.8603 FAR 0.00001
[1] TAR: 0.9284 FAR 0.00010
[1] TAR: 0.9732 FAR 0.00100
[1][1] time: 14.19   amloss: 14.054   reg_loss: 3.875
[1][11] time: 5.73   amloss: 14.992   reg_loss: 3.841
[1][21] time: 5.49   amloss: 22.382   reg_loss: 3.756
[1][31] time: 4.70   amloss: 32.082   reg_loss: 3.610
[1][41] time: 3.16   amloss: 28.846   reg_loss: 3.410
[1][51] time: 3.93   amloss: 28.779   reg_loss: 3.202
[1][61] time: 3.95   amloss: 24.922   reg_loss: 3.009
[1][71] time: 3.96   amloss: 20.103   reg_loss: 2.849
[1][81] time: 3.21   amloss: 16.399   reg_loss: 2.724
[1][91] time: 3.17   amloss: 15.480   reg_loss: 2.640
Testing...
[1] TAR: 0.0000 FAR 0.000
[1] TAR: 0.0002 FAR 0.000
[1] TAR: 0.0008 FAR 0.001

**Comment by seasonSH** [comment]
Well, actually I never tried to stop fine-tuning and continue again. It seems that the loss became unstable after you continue. I think the reason could be that not all variables are restored. Please make sure that all of them are restored.

**Comment by hustzeyu** [comment]
@xinliu111 ，how did you solve the problem?
